### DISCUSSION TITLE
Spatia/Translatable & Voyager

### DISCUSSION THREAD

**Discussion by mr-davit** [thedevdojo/voyager#5752] [Q&A]
I want to build multilingual site with voyager admin panel, I know how to use spatia when creating all controllers by my own, but got lost in voyager,

when I'm adding spatia related rules in model, I get properly formatted column in database

{"en":"Matrix"}, and when I add second language in this column manually,

{"en":"Matrix","ka":"მატრიცა"}, change language works in blade,

I just can't add second language in this column from "add new post" page

I had Idea to combine two input fields when adding some new items, like this,

'title' => ['en'=>  $request->title_en,
                'ka'=> $request->title_ka ],

but I don't know where to find this $request

I wonder maybe there is easier way to adapt spatia with voyager without losing voyagers's flexibility

Stackoverflow didn't answer my question but you can also [there ](https://stackoverflow.com/questions/75900888/how-can-i-use-laravel-voyager-admin-panel-with-spatia-translatable-package)

**Comment by youseffatouraee** [comment]
Voyager supports multiple languages for your models. To get started, you need to configure the languages.
I think you won't need any third-party libraries then.

Take a look at its doc:
https://voyager-docs.devdojo.com/core-concepts/multilanguage
